Module documentation template (#8)
kyma-project/template-repository: README.md

Reviewer: Maybe we could add information to the `Installation` section that if there are multiple installation options, just mention the recommended one and include others in a separate document. As far as I know, using kubectl to install a module from the latest release is the recommended option for Kyma modules and should be put in the main README. I'm not sure if it applies also to external providers' repos

> > **NOTE:** This is a general template that can be used for a project README.md. Except for the mandatory sections, use only those sections that suit your use case but keep the proposed section order.

Reviewer: ```suggestion
> **NOTE:** This is a general template that you can use for a project README.md. Except for the mandatory sections, use only those sections that suit your use case but keep the proposed section order.
```

>

> > Mandatory sections: 

Reviewer: Shouldn't `Installation` also be a mandatory section?

> - `Overview`
> - `Prerequisites`, if there are any requirements regarding hard- or software

> > - `Contributing` - do not change this! -

Reviewer: ```suggestion
> - `Contributing` - do not change this!
```
